src/rules.jl

```diff
@@ -103,4 +103,16 @@ true
 
 See also: [`frule`](@ref), [`@scalar_rule`](@ref)
 """
-rrule(::Any, ::Vararg{Any}; kwargs...) = nothing
+rrule(::Any, ::Vararg{Any}) = nothing
+
+# Manual fallback for keyword arguments. Usually this
+# would be generated by
+#
+#   rrule(::Any, ::Vararg{Any}; kwargs...) = nothing
+#
+# However - the fallback method is so hot that we want
+# to avoid any extra code that would be required to
+# have it package up the keyword arguments,
+# so we create this method manually.
```

Comment:
BlueStyle, like julia itself, wraps at 92 characters.
```suggestion
# Manual fallback for keyword arguments. Usually this would be generated by
# `rrule(::Any, ::Vararg{Any}; kwargs...) = nothing`
# However - the fallback method is so hot that we want to avoid any extra code that would be
# required to have it package up the keyword arguments,so we create this method manually.
```